### PR TITLE
GUI fixes

### PR DESCRIPTION
**Pull request by akortunov**
1) Fixes quick keys menu width calculation and make it non-transparent ([screenshot](http://i.imgur.com/P4PPVSr.jpg))
2) Removes "Cost/Change" and "Cost/Charge" labels from quick key magic selection dialog and make it non-tranparent ([screenshot](http://i.imgur.com/TMRD4bi.jpg))
3) Makes character selection popup in game load menu non-transparent ([screenshot](http://i.imgur.com/z1smMFQ.jpg)).

### CODE REVIEW

**Comment by uramer** [comment]
aren't those menus supposed to be transparent? At least I liked them that way. Good job on removing those silly bugs though!

**Comment by akortunov** [comment]
> aren't those menus supposed to be transparent?

No. Almost all widgets placed over other widgets are not transparent in vanilla game.

File: files/mygui/openmw_resources.xml [hunk 1]
```diff
@@ -114,6 +114,24 @@
         </Widget>
     </Resource>
 
+    <Resource type="ResourceLayout" name="MW_ComboBoxNoTransp" version="3.2.0">
```
Comment (scrawl) [inline files/mygui/openmw_resources.xml]: As far as I recall, ComboBox is only used once so you can just change the original skin instead.

Comment (akortunov) [inline files/mygui/openmw_resources.xml]: It is worth to create a separate PR for that?

Comment (scrawl) [inline files/mygui/openmw_resources.xml]: Sure.

Comment (akortunov) [inline files/mygui/openmw_resources.xml]: Just a note: combobox is also used in settings menu (e.g. in water tab).

**Pull request merged**